src/ui/pages/config/SinglePageSettingsFormContainer.js

```diff
@@ -22,7 +22,7 @@ export const mapStateToProps = state => ({
 export const mapDispatchToProps = (dispatch, ownProps) => ({
   onWillMount: () => {
     dispatch(fetchLanguages(noPagination, activeLangQueryString));
-    dispatch(fetchGroups(noPagination));
+    dispatch(fetchGroups());
```

Comment:
dont we want all groups fetched here without pagination?

Reply:
ah I see. Maybe it makes sense to rename `fetchGroups` to `fetchMyGroups` to be clearer?